API.md

```diff
@@ -117,12 +149,13 @@ Parameters
 ```
     documentId: String object containing document id (e.g. a DID)
     serviceId: String, representing the list of `file` objects that describe each file in the dataset
```

Comment:
This serviceId is the same as the serviceId from initialize, so I think the description should match.  

```
serviceId: String representing a service in the list of services in the DDO document
```

Reply:
@DMats,
I have replace `serviceID` description with the one from [serviceId ](https://github.com/oceanprotocol/docs/blob/feature/ddo_v4/content/concepts/did-ddo.md#services)

`serviceID: ID of the service the datatoken is attached to.`